src/main/java/net/whistlingfish/harmony/protocol/OAReplyParser.java

```diff
@@ -74,4 +83,8 @@ private Object parsePseudoJsonValue(String value) {
             return value;
         }
     }
+    
+    public boolean validResponseCode(String code){
+    	return validResponses.contains(code);
+    }
```

Comment:
I moved this here so message response parsers could control what a valid response is